### ISSUE TITLE
Enhancement request: more filters in one process

### ISSUE THREAD

**Issue by kseistrup** [ikalnytskyi/em#3]
It would be more useful if it was possible to add more filters to one `em` instance, e.g.:

``` sh
$ tail -f debug.log | em "ERROR" red "WARNING" yellow
```

or

``` sh
$ tail -f debug.log | em --red "ERROR" --yellow "WARNING"
```

**Comment by ikalnytskyi** [comment]
I plan to implement it, but currently you can achieve this functionality by using Unix pipes:

```
$ tail -f debug.log | em "ERROR" red | em "WARNING" yellow
```

**Comment by kseistrup** [comment]
Yes, I'm aware about piping, but it seems a little overkill to run another pipe for each color — hence my suggestion.

**Comment by ikalnytskyi** [comment]
> but it seems a little overkill to run another pipe for each color

Why? It's a Unix way. For instance we use `grep` this way too:

```
$ tail -f /path/to/log | grep "pattern1" | grep "pattern2"
```

Performance and memory usage are only reasons I want to implement more filters. :)
So stay in touch, it will be in next release.
